AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/commands/home/SetHomeCommand.java

```diff
@@ -49,7 +49,7 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
                 }
             }
 
-            if (atPlayer.canSetMoreHomes()) {
+            if (atPlayer.canSetMoreHomes() || (NewConfig.get().OVERWRITE_SETHOME.get() && atPlayer.hasHome(args[0]))) {
```

Comment:
What on earth, I thought I added my review here... really sorry :(

With respect to this, we may want to approach this similarly to the player doing `/movehome` instead of just `/sethome`.

Reply:
Doing that would be nonsenical as sethome does work just like movehome when not at the limit.
Solving this disparity is the whole point of this PR

Reply:
Oh, you're right there, my mistake.